src/scores/probability/crps_impl.py

```diff
@@ -174,35 +174,39 @@ def crps_cdf(
     is used as an index for predictive CDF values. Various techniques are used to
     interpolate CDF values between indexed thresholds.
 
-    Given
-        - a predictive CDF `fcst` indexed at thresholds by variable x,
-        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs),
-        - a `threshold_weight` array indexed by variable x,
+    Given:
+        - a predictive CDF `fcst` indexed at thresholds by variable x
+        - an observation in CDF form `obs_cdf` (i.e., :math:`obs\\_cdf_x = 0` if :math:`x < obs` and 1 if :math:`x >= obs`)
+        - a `threshold_weight` array indexed by variable x
 
     The threshold-weighted CRPS is given by:
-        `CRPS = integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x.
-    The usual CRPS is the threshold-weighted CRPS with `threshold_weight(x) = 1` for all x.
-    This can be decomposed into an over-forecast penalty
-        `integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x where x >= obs
-    and an under-forecast penalty
-        `integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x where x <= obs.
+        - :math:`CRPS = \\int{[threshold\\_weight_x \\times (fcst_x - obs\\_cdf_x)^2]}`, over all thresholds x.
```

Comment:
You could do something like
```suggestion
        - :math:`CRPS = \\int_{-\\infty}^{\\infty}\\text{threshold\_weight}(x)(\\text{fcst}(x)-\\text{obs\\_CDF}(x))^2 dx
```
With this notation, you want to make use of `\text` for worded variables.

An alternative approach would be to replace worded variables with a sensible letter and define what that letter is. below.

Reply:
Most of the stats literature uses an indicator function in the twCRPS definition. Not everyone in the met community is familiar with indicator functions, so it may be worth avoiding. I'm just mentioning this in case you want to cross-check this against twCRPS equations on the web.

Reply:
Updated, thanks.